sdk/api/metalava.txt

```diff
@@ -12,7 +12,7 @@ package com.mapbox.maps {
     method public void removeRendererSetupErrorListener(com.mapbox.maps.renderer.RendererSetupErrorListener rendererSetupErrorListener);
     method @com.mapbox.maps.MapboxExperimental public boolean removeWidget(com.mapbox.maps.renderer.widget.Widget widget);
     method public void setMaximumFps(int fps);
-    method public void setOnFpsChangedListener(com.mapbox.maps.renderer.OnFpsChangedListener listener);
```

Comment:
interesting that the binary compatibility validator didn't seem to care about the nullability changes, according to https://github.com/Kotlin/binary-compatibility-validator/issues/20

however, shall we consider the risk of breaking changes here?